llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1086,15 +1087,14 @@ void VPInstruction::print(raw_ostream &O, const Twine &Indent,
 
 void VPInstructionWithType::execute(VPTransformState &State) {
   State.setDebugLocFrom(getDebugLoc());
-  switch (getOpcode()) {
-  case Instruction::ZExt:
-  case Instruction::Trunc: {
+  if (isScalarCast()) {
```

Comment:
Needed because one of the recipes expanded to is Instruction::UIToFP. Happy to split this off to an NFC if wanted.